userguide/shutdown.rst

```diff
@@ -18,15 +17,19 @@ will negatively impact current users of the %brand% system.
    Shutdown Warning Message
 
 
-If a scrub or resilver is in progress when a shutdown is requested, an
-additional warning will ask if you wish to proceed. In this case, it
-is recommended to :guilabel:`Cancel` the shutdown request and to
-periodically run :command:`zpool status` from :ref:`Shell` until the
-scrub or resilver process is complete. Once complete, the shutdown
-request can be re-issued.
+.. If a scrub or resilver is in progress when a shutdown is requested, an
+   additional warning will ask if you wish to proceed. In this case, it
+   is recommended to :guilabel:`Cancel` the shutdown request and to
```

Comment:
This is confusing and complicated.  Let's just say "If a scrub or resilver is running, a warning is shown. Clicking :guilabel:`Cancel` is recommended. :command:`zpool status` can be run from the :ref:`Shell` to watch for the scrub or resilver to complete. Then the system can be shut down normally."